app/presenters/organisations/header_presenter.rb

```diff
@@ -0,0 +1,118 @@
+module Organisations
+  class HeaderPresenter
+    attr_reader :org
+
+    def initialize(organisation)
+      @org = organisation
+    end
+
+    def organisation_logo
+      component_data = {}
+      component_data[:name] = org.formatted_title.html_safe if org.formatted_title
+      component_data[:brand] = org.brand if org.brand
+      component_data[:crest] = org.crest if org.crest
+
+      if org.organisation_image_url
+        component_data[:image] = {
+          url: org.organisation_image_url,
+          alt_text: org.organisation_image_alt_text
+        }
+      end
+
+      component_data
+    end
+
+    def ordered_featured_links
+      links = []
+
+      if has_services_and_information_link?
+        see_more_link = {
+          text: "All #{org.title} services and information", # need to use acronym where available
+          path: "/government/organisations/#{slug}/services-information"
+        }
+      end
+
+      org.ordered_featured_links.each do |link|
+        links << {
+          text: link["title"],
+          path: link["href"]
+        }
+      end
+
+      {
+        small: small_header_links?,
+        brand: org.brand,
+        items: links,
+        see_more_link: see_more_link
+      }
+    end
+
+    def logo_wrapper_class
+      return "column-two-thirds" if small_header_links?
+      "column-one-third"
+    end
+
+    def link_wrapper_class
+      return "column-one-third" if small_header_links?
+      "column-two-thirds"
+    end
+
+    def translation_links
+      if org.translations.present? && org.translations.count > 1
+        links = []
+        org.translations.each do |link|
+          active = true if link["base_path"] == org.base_path
+
+          links << {
+            locale: link["locale"],
+            base_path: link["base_path"],
+            text: link["locale"],
+            active: active
+          }
+        end
+
+        {
+          brand: org.brand,
+          no_margin_top: true,
+          translations: links
+        }
+      end
+    end
+
+    def is_no_10?
+      true if org.organisation_type == "executive_office"
+    end
+
+  private
+
+    def slug
+      org.base_path.gsub("/government/organisations/", "")
+    end
+
+    def small_header_links?
```

Comment:
I feel like this method may be mis-named. I know it's being used to decide what layout we use for the header links, but it actually just returns whether something is news or not. Technically it could be used somewhere else for a different purpose later on. So it may be worth renaming it to make this clearer?

Reply:
I'll rename it later if it gets used for something more generic, but for now I wanted a method that clearly explained what it's for - organisations have too many things in them already that don't make sense unless you know lots about organisations.